utils/modify-payload-directory.sh

```diff
@@ -30,7 +33,7 @@ echo -e "${GREEN_FG}[\U2714] Updated name of payload directory to:${RESET}\
 if sudo docker ps | grep 'apache' > /dev/null; then
```

Comment:
Same question about `docker-compose`/`docker compose`.

Reply:
Resolved in [ba09100](https://github.com/cisagov/megazord-composition/pull/49/commits/ba09100eb8b671add3e275845b5d831bea1a43ee).